search/search-dev-tools/src/context.js

```diff
@@ -0,0 +1,3 @@
+import { createContext } from 'preact';
+
+export const SearchContext = createContext(null);
```

Comment:
Should have a clear linespace at the end of the file. This is usually settable within many IDEs to happen on save automatically.